Removed tokio-console from main.rs and Dockerfile. Logging still works as before.

param-jasani opened the pull request:
This PR is raised against the issue id - #222

lfbrehm reviewed (changes requested):
Now, you have added the `--cfg tokio_unstable` option back in, which we wanted to remove in the first place.

param-jasani commented:
> Now, you have added the `--cfg tokio_unstable` option back in, which we wanted to remove in the first place.

Sorry for the inconvenience on my side. I’ve fixed this again and verified it by running docker build; the image now builds successfully without any errors.


> One additional comment:
> 
> can you please in the end squash your commits into a single commit that complies with the [semantic commits](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716) standard. So something in the line of:
> 
> `chore: remove tokio-console from dataproxy`

I have squashed all the commits into one as you specified while following the semantic standard.

Thank you again to both of you for your invaluable guidance!!
